tests/test_analogue.py

```diff
@@ -117,11 +161,14 @@ def test_analogue_input(pytestconfig, board, config):
 @pytest.mark.uncollect_if(func=analogue_output_uncollect)
 @pytest.mark.parametrize(["board", "config"], list_configs())
 def test_analogue_output(pytestconfig, board, config):
-    features = get_config_features(board, config)
+    features_tmp = get_config_features(board, config)
+    features = features_tmp.copy()
 
     xsig_config = f'mc_analogue_output_{features["analogue_o"]}ch'
-    if board == "xk_316_mc" and features["tdm8"]:
-        xsig_config = "mc_analogue_output_2ch"
+    if board == "xk_316_mc":
+        features["samp_freqs"] = [max(features["samp_freqs"])] # only test against the analyser at max sample frequency when loopback is avalable
```

Comment:
Are we happy that we are no longer ever testing the analogue outputs at lower sample frequencies? This would have to be agreed with what we claim to test/support. It's definitely fine for the smoke level, but I'm unsure whether we should be testing all the frequencies when we have more time on nightly or weekend runs.

Reply:
rough estimate puts the loopback testing as taking 3h30min on nightly and 20h15min on weekend runs.
Are we willing to sacrifice test duration, number of fully tested configs or something else to account for the extra time?

Reply:
I would say to reduce test durations to fit the available time - analogue input and loopback tests should be longer (maybe significantly so) than analogue output tests.